client/src/store/historyStore/model/filtering.js

```diff
@@ -107,7 +122,7 @@ const validFilters = {
     visible: equals("visible", "visible", toBool),
     name: contains("name"),
     state: equals("state"),
-    tag: contains("tags", "tag"),
+    tag: contains("tags", "tag-contains", expandNameTag),
```

Comment:
Can you explain why we change to `tag-contains` here, I thought this only correlates to the key in the filter text?